src/components/Landing/Hero.tsx

```diff
@@ -1,36 +1,127 @@
 } from "@mantine/core";
 import styles from "./Hero.module.css";
 
+const images = [
+  {
+    src: "/images/hero/photo1.jpg",
+    title: "The life moments",
+    text: "Learn the art of analog photography",
+  },
+  {
+    src: "/images/hero/photo2.jpg",
+    title: "Timeless Shots",
+    text: "Discover the beauty of film photography",
+  },
+  {
+    src: "/images/hero/photo3.jpg",
+    title: "Retro Elegance",
+    text: "Master manual settings and composition",
+  },
+  {
+    src: "/images/hero/photo4.jpg",
+    title: "Unique Aesthetics",
+    text: "Create unique images with real grain",
+  },
+  {
+    src: "/images/hero/photo6.jpg",
+    title: "Keep Memories",
+    text: "Every photo tells a story",
+  },
+];
+
 export default function Hero() {
   const theme = useMantineTheme();
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const [fade, setFade] = useState(true);
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setFade(false);
+      setTimeout(() => {
+        setCurrentIndex((prevIndex) => (prevIndex + 1) % images.length);
+        setFade(true);
+      }, 300);
+    }, 5000);
+
+    return () => clearInterval(interval);
+  }, []);
+
+  const handlePrev = () => {
+    setFade(false);
+    setTimeout(() => {
+      setCurrentIndex(
+        (prevIndex) => (prevIndex - 1 + images.length) % images.length,
+      );
+      setFade(true);
+    }, 300);
+  };
+
+  const handleNext = () => {
+    setFade(false);
+    setTimeout(() => {
+      setCurrentIndex((prevIndex) => (prevIndex + 1) % images.length);
+      setFade(true);
+    }, 300);
+  };
+
   return (
     <div className={styles.hero}>
+      <div className={styles.controls}>
+        <button className={styles.controlButton} onClick={handlePrev}>
+          ◀
+        </button>
+        <button className={styles.controlButton} onClick={handleNext}>
+          ▶
+        </button>
+      </div>
+
       <div className={styles.imageWrapper}>
-        <Image
-          src="/images/hero/heroCamera.png"
-          alt="Animovaný obrázok"
-          className={styles.image}
-        />
+        {images.map((image, index) => (
+          <BackgroundImage
+            key={index}
+            src={image.src}
+            className={styles.image}
+            style={{ opacity: index === currentIndex ? 1 : 0 }}
+            radius={theme.radius.lg}
+          />
+        ))}
       </div>
 
       <Container>
-        <div className={styles.content}>
-          <Title order={1}>Don't be scared to capture ANAL(og) photos</Title>
-          <Text size="lg" mt="md">
-            We will teach you how to do it
-          </Text>
-          <Button size="lg" c="white" bg={theme.colors.orange[6]} mt="xl">
-            See our courses
-          </Button>
+        <div
+          className={styles.content}
+          style={{
+            opacity: fade ? 1 : 0,
+            transition: "opacity 0.5s ease-in-out",
+          }}
+        >
+          <div>
```

Comment:
use Box
in future if we want to reuse components it is going to be more convenient